ORK1Kit/ORK1Kit/Common/ORK1TaskViewController.m

```diff
@@ -748,7 +748,10 @@ - (NSArray *)managedResults {
         id <NSCopying> key = [self uniqueManagedKey:identifier index:idx];
         ORK1Result *result = _managedResults[key];
         NSAssert2(result, @"Result should not be nil for identifier %@ with key %@", identifier, key);
-        [results addObject:result];
+        if ([identifier isEqualToString:self.currentStepViewController.step.identifier] && self.currentStepViewController.step.saveOnArrival) {
```

Comment:
The empty if-condition reads a little weird, I think it would be cleaner to invert the if condition and eliminate the else: `if (!....) { [results addObject:result]; }` or if that ends up being too hard to read, could do `BOOL excludeFromResult = [identifier isEqualToString.....; if (!excludeFromResult) {  ... }`

Reply:
Agreed! I had a log statement to bookmark in the debugger and removed it and hadn't noticed I left an empty case. Thank you!